apps/fs/shell/Cargo.toml

```diff
@@ -8,6 +8,10 @@ authors = ["Yuekai Jia <equation618@gmail.com>"]
 
 [features]
 use_ramfs = ["axfs/myfs", "dep:axfs_vfs", "dep:axfs_ramfs", "dep:crate_interface"]
+
+# if needs irq, add "libax/irq"
+use-ramdisk = ["use_ramfs", "libax/use-ramdisk"]
```

Comment:
It is no need to enable `use_ramfs` when enabe `use-ram disk`.

Reply:
Okay